### PR TITLE
Views do not get recreated and are long lived

### PR DESCRIPTION
**Pull request by hakubo**
that seems to cause articles to be appended to a previous article

### CODE REVIEW

File: front/templates/main/article.hbs [hunk 1]
```diff
@@ -33,13 +33,13 @@
 					{{/each}}
 				{{/collapsible-menu}}
 			{{/if}}
-			{{#view 'article-content' class='mw-content'}}
+			{{#article-content class='mw-content'}}
```
Comment (hakubo) [inline front/templates/main/article.hbs]: @kenkouot 
Actually to be honest I have no idea what is the benefit of having this as  a component/view?
Can't this be just a <div class='mw-content'> ?

Comment (kenkouot) [inline front/templates/main/article.hbs]: Agreed, there might have been a point some time ago, but I vote for simplicity!

**Comment by rogatty** [comment]
@hakubo: how is this one going? Want to remove article-content?

**Comment by hakubo** [comment]
I am going to close this now.
I would go for removing article content component/view

**Pull request closed without merging**